pydash/pydash_web/controller/dashboards.py

```diff
@@ -0,0 +1,45 @@
+"""
+Manages the lookup and returning of dashboard information for a certain user.
+
+Currently only returns static mock data.
+"""
+
+from flask import jsonify
+
+
+# Currently returns mock-data.
+def get_dashboard(dashboard_id):
+
+    return jsonify(__read_json_mock_data(0)), 200
+
+
+# Currently returns mock-data.
+def get_dashboards():
+    dashboards = [__read_json_mock_data(0), __read_json_mock_data(1)]
+    return jsonify(dashboards), 200
+
+
+# Returns some static json mock data, for testing purposes
+def __read_json_mock_data(index):
```

Comment:
You could move the `index` call to outside of this function; i.e. `__read_json_mock_data()[index]`.

I'd also suggest renaming this function to `_json_mock_data`: double underscores are not used in idiomatic python; with a single underscore something is not really private but it is as private as we want it to be at the most.